include/pclomp/ndt_omp.h

```diff
@@ -228,7 +228,7 @@ namespace pclomp
 		}
 
 		/** \brief Return the transformation array */
-		inline const std::vector<Eigen::Matrix4f>
+		inline const std::vector<Eigen::Matrix4f, Eigen::aligned_allocator<Eigen::Matrix4f>>
```

Comment:
Could you link Eigen's documents as well?
Since I think we'll submit a PR to upstream, we need some references to explain to the maintainer.

Reply:
I guess it has something to do with this issue https://github.com/isl-org/Open3D/issues/653